lib/config/default.js

```diff
@@ -1382,7 +1383,13 @@ defaultConfig.definition = () => ({
         default: true
       }
     }
-  }
+  },
+  /**
+   * Stanza that contains all keys to disable 3rd party package instrumentation(i.e. mongodb, pg, redis, etc)
+   * Note: Disabling a given 3rd party library may affect the instrumentation of 3rd party libraries used after
+   * the disabled library.
+   */
+  instrumentation: pkgInstrumentation
```

Comment:
```suggestion
  /**
   * Stanza that contains all keys to disable 3rd party package instrumentation(i.e. mongodb, pg, redis, etc)
   * Note: Disabling a given 3rd party library may affect the instrumentation of 3rd party libraries used by
   * the disabled library.
   */
  instrumentation: pkgInstrumentation
```

Reply:
i meant to say any instrumented libraries used after the disabled instrumented, it's not really by the disabled library